src/internal/kopia/upload.go

```diff
@@ -183,6 +179,30 @@ func (cp *corsoProgress) FinishedFile(relativePath string, err error) {
 	}
 
 	cp.deets.AddFolders(folders)
```

Comment:
Following up on the slack discussion - where we've changed this to `AddFoldersForItem` in `main` . I think we should just defer the `AddFoldersForItem` for an item that is being sourced from the base snapshot to when it is being "merged into details"

Reply:
yea, that makes the most sense. A bit of a bummer cause I wanted to keep the code for building the set of directories limited to as few places as possible. Guess I'll move it to a helper function somewhere